### PR TITLE
Add api version 2019-03-01 for Microsoft.Compute

### PR DESCRIPTION
**Pull request by ashintre**
Add api version 2019-03-01 for Microsoft.Compute

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5613 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5613/commits/26ba0025a92d8ceacf5613d8d9d666ad428771ad

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
You will need to add a new entry in the [README](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md) for this API version.
Also, are the only new changes in this API version in the second commit?
Finally, has this API version been deployed?

**Comment by ashintre** [comment]
> You will need to add a new entry in the [README](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md) for this API version.
> Also, are the only new changes in this API version in the second commit?
> Finally, has this API version been deployed?

Yes, this API version has already

**Comment by ashintre** [comment]
> You will need to add a new entry in the [README](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md) for this API version.
> Also, are the only new changes in this API version in the second commit?
> Finally, has this API version been deployed?

The API version has been deployed to all regions. There are a couple of other properties that have also been added in the new version, but will be added on top of this PR by other engineers.

**Comment by jhendrixMSFT** [comment]
Got it, I've added the `DoNotMerge` label for now.  Please let me know when all of the remaining content has been added.

**Comment by dsgouda** [comment]
startbuild

**Comment by ashintre** [comment]
> Got it, I've added the `DoNotMerge` label for now. Please let me know when all of the remaining content has been added.

Joel, what I meant was that they are planning to add the content in separate PRs after this is merged.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -0,0 +1,8387 @@
+{
```
Comment (KrisBash) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: In the future, please consider copying over the last production API version as the first commit of the PR. It makes viewing the deltas viable in GH.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 2]
```diff
@@ -0,0 +1,8387 @@
...
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/redeploy": {
+      "post": {
+        "tags": [
+          "VirtualMachines"
+
```
Comment (KrisBash) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Spacing

Comment (KrisBash) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: As a general rule, string enums are more futureproof than boolean, in that you can add additional enum values without a breaking type change.

Comment (KrisBash) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Please confirm that a linked access check will be defined for this

**Review by KrisBash — APPROVED** [review]
Signing off with comments. Some hygiene items to clean up but no blockers.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**